src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -785,9 +784,9 @@ uct_ib_mlx5_devx_check_odp(uct_ib_mlx5_md_t *md,
     return UCS_OK;
 }
 
-static ucs_status_t
-uct_ib_mlx5_devx_query_lag(uct_ib_mlx5_md_t *md, uint8_t *state,
-                           uct_ib_port_select_mode_t *port_select_mode)
+static void
```

Comment:
return ucs_status_t, or uct_ib_port_select_mode_t (UCT_IB_MLX5_LAG_INVALID_MODE if the command fails)
seems that if the command fails, currently the value of *port_select_mode is undefined